docs/topics/getting-started-ultrahook-setup.adoc

```diff
@@ -0,0 +1,37 @@
+[[ultrahook-setup]]
+= Trigger Builds Automatically with UltraHook
+
+To configure your {OpenShiftLocal} to automatically trigger a build when a new commit is made to GitHub, we will use the Ultrahook's proxy cloud application. This application exposes a public endpoint that Github will call when an event (commit pushed example) is issued. If the Ultrahook client is running on your local machine, then it will forward the HTTP request to an OpenShift Build Trigger endpoint in order to initiate the start of your booster's project build:
```

Comment:
What's "commit pushed example"? I'd replace "This application exposes a public endpoint that Github will call when an event (commit pushed example) is issued" with "This application exposes a public endpoint that GitHub will call when a commit is pushed".